README.md

```diff
@@ -1,45 +1,191 @@
 # module-manager
 
-This repository offers an operator and relevant packages to perform installation, update and uninstallation of helm charts.
-The packages could be used independent of the operator.
+The [module-manager](#module-manager) plays an important part in the modularization ecosystem to handle installation of resources from control-plane to runtime clusters. 
+For more information, refer to the [architecture summary](https://github.com/kyma-project/lifecycle-manager#architecture).
+This repository offers an **_operator_** and relevant **_library packages_** to perform reconciliation of helm chart resources, configurations and custom resource state handling.
+The **_library package_** could be used independently of the operator.
 
-**Disclaimer**: This repository is still in development and implementation details can change quickly.
+### Content
 
-## Local setup
+* [Operator](#operator)
+* [Manifest library](#manifest-library) 
+* [Local setup](#local-setup)
+* [Cluster setup](#cluster-setup)
+* [Contribution](#contribution)
+* [Versioning and releasing](#versioning-and-releasing)
+* [Next steps](#next-steps)
 
-- Spin up a k3d cluster (other types like kind are also supported)
-- From the **api** folder run: 
-`make install`
-- From the **operator** folder run
-  `make build` 
-followed by
-  `make run`
-to run the operator
-- Config your own **kubeconfig** by setting the **r.Config** variable inside */operator/controllers/manifest_controller.go*
+**Disclaimer**: This repository is still in development and implementation details could change rapidly.
```

Comment:
I think we should add a stability section similar to https://github.com/kyma-project/lifecycle-manager#stability